Reconstruction/RecTracker/src/components/FastHitFilterTool.h

```diff
@@ -0,0 +1,42 @@
+#ifndef RECTRACKER_DOUBLETCREATIONTOOL_H
+#define RECTRACKER_DOUBLETCREATIONTOOL_H
+
+// from Gaudi
+#include "GaudiAlg/GaudiTool.h"
+
+// FCCSW
+#include "FWCore/DataHandle.h"
+#include "RecInterface/ITrackSeedingTool.h"
+#include "RecInterface/IHitFilterTool.h"
+#include "RecInterface/ILayerGraphTool.h"
+#include "datamodel/PositionedTrackHitCollection.h"
+#include "datamodel/TrackHitCollection.h"
+#include "datamodel/TrackStateCollection.h"
+
+#include "tricktrack/SpacePoint.h"
+#include "tricktrack/CMGraph.h"
+#include "tricktrack/TrackingRegion.h"
+#include "tricktrack/HitChainMaker.h"
+
+#include <map>
+
```

Comment:
missing doxy.
Also since it's a component, its name should suggest it is used for TrackHits.